cmd/stack_yaml_lint.go

```diff
@@ -49,7 +49,7 @@ func (stackDetails *StackYaml) validateYaml(rootConfig *RootCommandConfig, stack
 	}
 
 	stackLintErrorCount += stackDetails.checkDescLength(rootConfig.LoggingConfig)
-	stackLintErrorCount += stackDetails.checkLicense(rootConfig.LoggingConfig)
+	stackLintErrorCount += stackDetails.checkLicense(rootConfig)
```

Comment:
By only using `log *LoggingConfig` in the functions below, you can remove this change